packages/generator/src/commonTemplateHelper.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Given an individual type or an array of types in the format Array\<Foo | Baa\>
+ * will return either the type prefixed by the namespace, or the Array with each type prefixed
+ * eg. Array\<types.Foo | types.Baa\>
+ *
+ * @param content - to be parsed for types to prefix with a namespace
+ * @param namespace - to be prefixed to types
+ * @returns the content prefixed with the namespace
+ */
+export function addNamespace(content: string, namespace: string): string {
```

Comment:
Looking at the templates, is the `namespace` provided ever not `"types"`? Will it ever be different? The namespace could be hard-coded or made an optional parameter (although the latter is a small bit of extra work due to Handlebars).

Reply:
Originally had it that way.  Actually talking it over with Daniel we changed it to be this way.  Basically, I think this is a typescript/javascript utility function and others might want to prefix it differently, like DTO or something.  But more than that I wanted the context of what is being added in the handlebars template and not in the javascript code.